### PR TITLE
Model Setup, Model Name: Add extra chars

### PR DESCRIPTION
**Pull request by olliw42**
PR complementing issue https://github.com/EdgeTX/edgetx/issues/431

adds more extra chars to the text edit field for model name.

it does so only for model name. If it should happen that similar extension should be desired elsewhere, it would be easy to add, and maybe the extra char string defined more globally, but all this and such should be part of a separate PR, IMHO.

tested on tx16s

### CODE REVIEW

File: radio/src/gui/colorlcd/model_setup.cpp [hunk 1]
```diff
@@ -1033,7 +1033,8 @@ void ModelSetupPage::build(FormWindow * window)
 
   // Model name
   new StaticText(window, grid.getLabelSlot(), STR_MODELNAME);
-  auto text = new RadioTextEdit(window, grid.getFieldSlot(), g_model.header.name, sizeof(g_model.header.name));
+  auto text = new RadioTextEdit(window, grid.getFieldSlot(), g_model.header.name, sizeof(g_model.header.name),
+          0, "_-.,:;<=>");
```
Comment (raphaelcoeffic) [inline radio/src/gui/colorlcd/model_setup.cpp:1037]: Can we please have these in some kind of definitions? Ideally, some `const static`.

Comment (olliw42) [reply radio/src/gui/colorlcd/model_setup.cpp:1037]: we of course could
I would need some suggestion though
- where/which file should it be placed?
- recommendation for name of it?

Comment (raphaelcoeffic) [reply radio/src/gui/colorlcd/model_setup.cpp:1037]: Very good question! Frankly, I have no clue for now, inside the same file would be ok for now, but we will need some restructuring soon, this code is a mess :-D

Comment (olliw42) [reply radio/src/gui/colorlcd/model_setup.cpp:1037]: that's why I just put it here for now ... for as long as it is not needed/required elsewhere it's not so much of  harm 
(and efficiency wise it doesn't matter anyways, the compiler does its optimization job)

Comment (olliw42) [reply radio/src/gui/colorlcd/model_setup.cpp]: ok, so did something
have to go now however, day work

Comment (raphaelcoeffic) [reply radio/src/gui/colorlcd/model_setup.cpp]: Thx! Looks ok to me!

Comment (olliw42) [reply radio/src/gui/colorlcd/model_setup.cpp]: great, so I mark it as resolved

**Pull request merged**